apps/src/templates/Congrats.story.jsx

```diff
@@ -1,138 +1,117 @@
 import React from 'react';
 import Congrats from './Congrats';
-import { Provider } from 'react-redux';
-import { combineReducers, createStore } from 'redux';
-import responsive from '../code-studio/responsiveRedux';
 
 export default storybook => {
-  const store = createStore(combineReducers({responsive}));
   return storybook
     .storiesOf('Congrats', module)
+    .withReduxStore()
     .addStoryTable([
       {
         name: 'Congrats - Applab, signed out',
         description: `Congrats component if Applab tutorial completed`,
         story: () => (
-          <Provider store={store}>
             <Congrats
               completedTutorialType="applab"
               isRtl={false}
               userType="signedOut"
               isEnglish={true}
             />
```

Comment:
Shouldn't there be indentation changes throughout here, with the Provider removed?

Reply:
I think we accept either 4 or 2 spaces for line continuations?  (Style guide says 4 for continuations, but 2 seems to be more common in the codebase.)

I'll update if you want!